api/public_stash_tab_subscription.go

```diff
@@ -80,6 +83,7 @@ func (s *PublicStashTabSubscription) run(firstChangeId string) {
 
 			if len(tabs.Stashes) > 0 {
 				s.Channel <- PublicStashTabSubscriptionResult{
+					ChangeId:        nextChangeId,
```

Comment:
@abesto That makes sense. In that case, I think you just have a bug here. See line 82. At this point, `nextChangeId` has already been set to `tabs.NextChangeId`. I think you can just move line 82 to below this this `if` block.

Reply:
Indeed, nice catch. Lemme fix that.